src/YGPushButton.cc

```diff
@@ -9,6 +9,7 @@
 #include "YGUtils.h"
 #include "YGWidget.h"
 #include <string.h>
+#include <libgen.h>
```

Comment:
in general looks OK. Just maybe this libgen adds additional dependency that is not addressed here?